base/client/src/main/java/dev/rico/client/projector/Projector.java

```diff
@@ -33,9 +34,7 @@
 
     ControllerProxy<? extends ManagedUiModel> getControllerProxy();
 
-    @Deprecated
```

Comment:
IIRC the @Deprecated were introduced by @giftkugel and @hendrikebbers to mark this implementation as a "temporary solution". In that case we should not remove it and we need to discuss if we would build further impl on top of it :)
WDYT @giftkugel and @hendrikebbers ?

Reply:
IIRC it was deprecated because I could not remember the reason why I introduced the modelToNodeMap. Now I do. It was, because if we don't have a modelToNodeMap, the projector recreates javafx nodes if we take a model out of the model tree and later insert it back into the model tree. That is an unwanted behavior. With the map the corresponding javafx node is not recreated but taken out of the map and inserted into the java fx node tree again.